modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -505,24 +509,10 @@ private void _configureClientExtensionTasks(
 
 		assembleClientExtensionTaskProvider.configure(
 			copy -> {
-				copy.from(
-					createClientExtensionConfigTaskProvider,
-					spec -> spec.eachFile(
-						fileCopyDetails -> {
-							File buildDir = project.getBuildDir();
-
-							File file = fileCopyDetails.getFile();
-
-							Path buildPath = buildDir.toPath();
-
-							Path relativePath = buildPath.relativize(
-								file.toPath());
-
-							fileCopyDetails.setRelativePath(
-								new RelativePath(
-									false, relativePath.toString()));
-						}));
-				copy.into(new File(project.getBuildDir(), "clientExtension"));
+				copy.dependsOn(CREATE_CLIENT_EXTENSION_CONFIG_TASK_NAME);
```

Comment:
You can't remove the FileCopyDetails stuff.  Otherwise gradle flattens the file relative paths.